### DISCUSSION TITLE
Question 4: Can you add a current date to a default? [LH]

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#745] [Q&A] [answered]
Description: Conservation added a new task default (“Review for attachment to catalog module”). Is there a way to have it input the current date when a new record is created?

**Comment by rondlg** [comment, marked as answer]
Response: We are not aware of a direct way to insert current date, but will reach out to Axiell. In the meantime, create and use, as an insert default, a [Task Template](https://help.emu.axiell.com/v5.1/en/Topics/Common/How%20to%20create%20and%20use%20a%20Task%20Template.htm?Highlight=Task%20Template)

**Comment by fmjjones** [comment]
Questions submitted to Hornbill SR00160437. 
Response from Axiell:
Unfortunately, EMu doesn't support the auto-fill date feature. However, there is a shortcut to insert today's date i.e. ctrl+;. For more information, please visit [Date and time fields](https://help.emu.axiell.com/v5.1/en/Topics/Common/Date%20and%20time%20fields.htm?Highlight=date%20shortcut). Please feel free to reach out for any assistance. We are looking forward to hearing back from you. Thank you.
